app/assets/stylesheets/user-account.css.scss

```diff
@@ -1,14 +1,18 @@
 @import "colors";
 
+#user-des{
+  margin-top: 55px;
+}
+
 .profile-pix{
   width: 70px;
   height: auto;
   padding-top: 10px;
-  z-index: 2000;
+  z-index: 150;
+  position: relative;
```

Comment:
Positive relative made it work
 On 5 Feb 2015 09:22, "Paul" <notifications@github.com> wrote:

> In app/assets/stylesheets/user-account.css.scss
> <https://github.com/gduck/fritter/pull/79#discussion_r24136680>:
>
> >  .profile-pix{
> >    width: 70px;
> >    height: auto;
> >    padding-top: 10px;
> > -  z-index: 2000;
> > +  z-index: 150;
> > +  position: relative;
>
> So, that's what makes it work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gduck/fritter/pull/79/files#r24136680>.
>